packages/core/admin/admin/src/pages/AuthPage/utils/forms.js

```diff
@@ -76,7 +76,7 @@ const forms = {
     fieldsToDisable: [],
     fieldsToOmit: ['confirmPassword', 'news'],
     schema: yup.object().shape({
-      firstname: yup.string().required(translatedErrors.required),
+      firstname: yup.string().trim().required(translatedErrors.required),
```

Comment:
I think this comes from a community PR I reviewed, this should be committed here 🤔

Reply:
yes sorry now it is on sync